src/frontend/screens/PhotosModal/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useMemo, useRef, useState } from "react";
+import { Dimensions, View, StyleSheet } from "react-native";
+import { NavigationState, TabView, Route } from "react-native-tab-view";
+
+import IconButton from "../../sharedComponents/IconButton";
+import Button from "../../sharedComponents/Button";
+import { CloseIcon, DeleteIcon } from "../../sharedComponents/icons";
+import { filterPhotosFromAttachments } from "../../lib/utils";
+import { useObservation } from "../../hooks/useObservation";
+import { useDraftObservation } from "../../hooks/useDraftObservation";
+import PhotoView from "../../sharedComponents/PhotoView";
+import api from "../../api";
+import type { NavigationProp } from "../../types";
+import { useBottomSheetModal } from "../../sharedComponents/BottomSheetModal";
+import { ConfirmDeleteModal } from "./ConfirmDeleteModal";
+import Text from "../../sharedComponents/Text";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { TouchableNativeFeedback } from "../../sharedComponents/Touchables/index.ios";
+
+interface customRoute extends Route {
+  uri?: string;
+  error?: boolean;
+  capturing?: boolean;
+}
+
+const m = defineMessages({
+  deleteImage: {
+    id: "screens.PhotosModal",
+    defaultMessage: "Delete Image",
+  },
+});
+
+const PhotosModal = ({ navigation }: { navigation: NavigationProp }) => {
```

Comment:
spent some time trying this and i think this shouldn't be a "modal" in the technical sense. Modals inherently introduce navigation functionality by being part of the navigation stack, which doesn't make sense for features like this. A heuristic I use is "would it make sense to ever be able to deeplink to this modal sheet?" In this case, not really. You'll also notice that with the current modal sheet implementation, the background content flickers because there's a transition when the sheet comes up, which especially looks odd on this screen.

There may be some other parts of the app using the modal sheet that also fall into this trap (maybe not as evidently), but i think it'd be worth creating a variation of the modal sheet that doesn't have any navigation implications. I have a WIP local branch that builds off this one that implements an identical bottom sheet (with some enhancements), so could be something that we could incorporate into this PR of as a follow up, depending on timing

Reply:
Let discuss in team meeting, Im not sure I fully understand your logic. I realize that adding the navigation prop is unnecessary here. I pass the prop a child, but I can just use the hook. If I do that, I can just avoid that navigation prop completely

Reply:
after discussion and rethinking this, it'd probably be enough to update the modal sheet component to just not have the custom background (think adding a prop and some conditional code will do the trick). didn't realize this wasn't being connected to the navigation stack so i think my concerns there don't apply.

we probably will have to create the more generic bottom sheet at some point but not essential for the time being

Reply:
Yeah, also on during that discussion I realized that I pass the navigation prop AND use it, but its not part of the stack so Im confused as to where it is getting the navigation prop. In my understanding, that prop was only passed to stack components, not all components

Reply:
Nevermind, I realize that the photos Modal which shows the photo as on overlay (not the bottom sheet) is on the stack, and Im passing that navigation prop to the bottom sheet!